tests/unit/providers/test_otari_provider.py

```
@@ -403,3 +411,266 @@ async def test_otari_aresponses_without_response_format() -> None:
+    ]
+
+    async def _mock_aiter_lines() -> AsyncIterator[str]:
+        for line in sse_lines:
+            yield line
+
+    mock_response = MagicMock(status_code=200)
+    mock_response.aiter_lines = _mock_aiter_lines
+
+    events = []
+    async for event in provider._stream_anthropic_events(mock_response):
+        events.append(event)
```

Ruff (0.15.15)

[warning] Use an async list comprehension to create a transformed list

(PERF401)